R/common.R

```diff
@@ -988,3 +988,77 @@ findBestAlternateScenarios <- function(dfAlternateFeat,
   dfOptResult
 }
 
+#' @title
+#' Generate ROC curve for a dataset.
+#' @description Generates ROC curve and AUC for Sensitivity/Specificity or 
+#' Precision/Recall.
+#' @param predictions A vector of predictions from a machine learning model.
+#' @param labels A vector of the true labels. Must be the same length as 
+#' predictions.
+#' @param aucType Can be "SS" or "PR". Defaults to SS.
+#' @param plotFlg Binary value controlling plots Defaults to FALSE (no).
+#' @return auc Integral AUC of ROC plot.
+#'
```

Comment:
Great job on this function, Mike! Do you mind putting the type of values expected for this params (and the return)? We haven't standardized completely, but would love to. Example:

- @param aucType A string. Indicates AU_ROC or AU_PR and can be "SS" or "PR". Defaults to SS.